lib/ci-helper.ts

```diff
@@ -144,6 +144,12 @@ export class CIHelper {
             return false;
         }
 
+        const commitsInPu: Set<string> = new Set<string>(
+            (await git(["rev-list", "--no-merges",
+                        "^refs/remotes/upstream/maint~100",
```

Comment:
Should `refs/heads/maint` be added to the git fetch at the start of this method?

Reply:
Good point!